packages/cadl-ranch-specs/http/versioning/removed/main.tsp

```diff
@@ -32,12 +34,17 @@ namespace Versioning.Removed;
  */
 enum Versions {
   /**
-   * The version v1.
+   * The original version v1.
    */
   v1: "v1",
 
   /**
-   * The version v2.
+   * The beta version.
+   */
+  beta: "beta",
```

Comment:
in our tests we generally use `preview` instead of `beta`. The preview is also attached with a specific version. Would this be `v3Preview`?

Reply:
The beta version is essentially a transitional version between V1 and V2, so renaming it to v2preview is more appropriate.  
 Fixed